core-bundle/assets/scripts/core.js

```diff
@@ -1668,6 +1668,12 @@ window.Backend =
 				}
 			},
 			clickEvent = function(e) {
+				const name = e.target.nodeName.toLowerCase();
+
+				if ('img' === name || 'a' === name || 'button' === name) {
+					return;
+				}
```

Comment:
Wondering if something like this is better as it'd be more typesafe but would work in both cases

```js
if (
    e.target instanceof HTMLImageElement || 
    e.target instanceof HTMLAnchorElement || 
    e.target instanceof HTMLButtonElement
) {
    return;
}
```

Reply:
Changed in 5595303.